OSDOCS-13310 Remove unverified perfscale info from ROSA docs (#88263)
openshift/openshift-docs: rosa_planning/rosa-hcp-prereqs.adoc

> :_mod-docs-content-type: ASSEMBLY

Reviewer: Preview link: https://88263--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs#rosa-sts-aws-prereqs

Reviewer: OR maybe this module is not published yet? Am I in the wrong doc set. Is HCP a new doc set?

Author: The ROSA docs are being split into separate Classic and HCP docs; the preview link above is for the Classic docs (which do currently still contain some HCP things that are not yet split)

> include::_attributes/attributes-openshift-dedicated.adoc[]

Reviewer: I cannot see this content in any of the provided links. I'll continue to review the module, but the module should be included in either the ROSA or HCP docs?

:context: rosa-hcp-aws-prereqs
[id="rosa-hcp-prereqs"]
= AWS prerequisites for {hcp-title}

toc::[]

> {hcp-title-first} provides a model that Red Hat hosts the control plane and uses your AWS account to deploy clusters.

Reviewer: ```suggestion
{hcp-title-first} provides a model that Red{nbsp}Hat hosts the control plane and uses your AWS account to deploy clusters.
```

Some people have said they experienced a bug where "Red" and "Hat" merge.

Author: Thanks, updating

Ensure that the following AWS prerequisites are met before installing ROSA with STS.

[IMPORTANT]
====

> When you create a ROSA cluster using AWS STS, an associated AWS OpenID Connect (OIDC) identity provider is created as well. This OIDC provider configuration relies on a public key that is located in the `us-east-1` AWS region. Customers with AWS SCPs must allow the use of the `us-east-1` AWS region, even if these clusters are deployed in a different region.

Reviewer: ROSA > {rosa-classic}?

Author: Changed to "{hcp-title}"

> When you create a ROSA cluster using AWS STS, an associated AWS OpenID Connect (OIDC) identity provider is created as well. This OIDC provider configuration relies on a public key that is located in the `us-east-1` AWS region. Customers with AWS SCPs must allow the use of the `us-east-1` AWS region, even if these clusters are deployed in a different region.

Reviewer: ```suggestion
When you create a ROSA cluster by using AWS STS, an associated AWS OpenID Connect (OIDC) identity provider is also created. This OIDC provider configuration relies on a public key that is located in the `us-east-1` AWS region. Customers with AWS SCPs must allow the use of the `us-east-1` AWS region, even if these clusters are deployed in a different region.
```

Expand SCP?
